### PR TITLE
feat: initial scaffolding for NodeJS

### PR DESCRIPTION
**Pull request by llorllale**
Initial scaffolding for NodeJS support.

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

File: cmd/aries-js-worker/app.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
```
Comment (troyronda) [inline cmd/aries-js-worker/app.js]: should we make the filename more explicit about being a node app?

Comment (llorllale) [reply cmd/aries-js-worker/app.js]: isn't "app" the convention?

File: cmd/aries-js-worker/aries.js [hunk 2]
```diff
@@ -4,18 +4,15 @@ Copyright SecureKey Technologies Inc. All Rights Reserved.
 SPDX-License-Identifier: Apache-2.0
 */
 
+'use strict'
+
+const _isNodeJS = typeof module !== 'undefined' && module.exports
```
Comment (troyronda) [inline cmd/aries-js-worker/aries.js]: is this variable name global to the containing page?

File: cmd/aries-js-worker/app.js (hunk 1 above)
Comment (llorllale) [reply cmd/aries-js-worker/app.js]: https://nodejs.org/en/docs/guides/getting-started-guide/

File: cmd/aries-js-worker/aries.js (hunk 2 above)
Comment (llorllale) [reply cmd/aries-js-worker/aries.js]: done

File: cmd/aries-js-worker/app.js (hunk 1 above)
Comment (troyronda) [reply cmd/aries-js-worker/app.js]: We are showing both browser and node.js samples aren't we?

Comment (troyronda) [reply cmd/aries-js-worker/app.js]: we should revisit once we have npm packaging. perhaps two subdirectories for the samples would make sense.

**Pull request merged**